sbin/create_genesis.sh

```diff
@@ -1,19 +1,28 @@
 #!/bin/bash
-# Check that the dotenv exists.
-ENV=".genesis.env"
-if ! test -f $ENV; then
-    echo "Expected dotenv at $ENV (does not exist)."
+
+# TODO: can we get rid of this somehow?
+# currently the local sbin paths are relative to the project root
+SBIN=$(dirname "$(readlink -f "$0")")
+SBIN="`cd "$SBIN"; pwd`"
+ROOT_DIR=$SBIN/..
+
+# Check that the all required dotenv files exists.
+PATHS_ENV=".paths.env"
+if ! test -f "$PATHS_ENV"; then
+    echo "Expected dotenv at $PATHS_ENV (does not exist)."
     exit
 fi
-echo "Using dotenv: $ENV"
-. $ENV
-
-if [ ! -d "$OPS_DIR" ]; then
-    SBIN=`dirname $0`
-    SBIN="`cd "$SBIN"; pwd`"
-    . $SBIN/configure.sh
-    OPS_DIR="`cd "$OPS_DIR"; pwd`"
+echo "Using paths dotenv: $PATHS_ENV"
+. $PATHS_ENV
+
+GENESIS_ENV=".genesis.env"
+if ! test -f "$GENESIS_ENV"; then
+    echo "Expected dotenv at $GENESIS_ENV (does not exist)."
+    exit
 fi
+echo "Using dotenv: $GENESIS_ENV"
+. $GENESIS_ENV
+
 echo "Using $OPS_DIR as ops directory."
```

Comment:
this is deleted though..?

Reply:
can you explain, will this be removed in other PR?